content/en/docs/Tasks/creating-an-update-graph.md

```diff
@@ -68,6 +68,22 @@ or contain bugs.
 
 Skipped CSVs do not need to be present in a catalog or set of manifests prior to adding to a catalog.
 
+#### replaces and skips example
```

Comment:
Nit: Capitalize like we do for all other sub header titles
```suggestion
#### Replaces and skips example
```
Or similar to what we have below:
```suggestion
#### Example for replaces and skips
```

Reply:
Done.